### ISSUE TITLE
[Feature] Compatibility with Create's train portals 

### ISSUE THREAD

**Issue by xbxbxb123** [SuperMartijn642/Wormhole#43]
The Create mod allows train tracks to go through Nether portals. This is a very fun feature, that I would really enjoy added to this mod's portals as well. Apologies if this feature already exists, I didn't see anything about it anywhere.

**Comment by SuperMartijn642** [comment]
I don't think this is really possible as the portals in Wormhole are one-way, so there's no way for me to make sure there's somewhere for the train to go on the other side

**Comment by xbxbxb123** [comment]
That's unfortunate. It would be nice if this was combined with that one request to only allow portals to connect to other portals.

**Comment by xbxbxb123** [comment]
Here's a link to that issue: 
https://github.com/SuperMartijn642/Wormhole/issues/18
